app/HMS/User/ProfileManager.php

```diff
@@ -86,4 +87,66 @@ public function create(User $user, string $address1, ?string $address2, ?string
 
         return $user;
     }
+
+    /**
+     * update the user form a form request.
+     * @param  User    $user    user to update
+     * @param  Illuminate\Http\Request $request
+     * @return User
+     */
+    public function updateUserProfileFromRequest(User $user, Request $request)
```

Comment:
Has this request been sanitized before it gets here? I forget laravels functionings.